cupy/cuda/cusparse.pyx

```diff
@@ -532,6 +554,54 @@ cpdef dcsr2dense(
     check_status(status)
 
 
+cpdef snnz_compress(
+        size_t handle, int m, size_t descr,
+        size_t values, size_t rowPtr, size_t nnzPerRow,
+        size_t nnzTotal, float tol):
+    status = cusparseSnnz_compress(
+        <Handle>handle, m, <const MatDescr>descr,
+        <const float *>values, <const int *>rowPtr, <int *>nnzPerRow,
+        <int *>nnzTotal, tol)
+    check_status(status)
+
+
+cpdef dnnz_compress(
+        size_t handle, int m, size_t descr,
+        size_t values, size_t rowPtr, size_t nnzPerRow,
+        size_t nnzTotal, double tol):
+    status = cusparseDnnz_compress(
+        <Handle>handle, m, <const MatDescr>descr,
+        <const double *>values, <const int *>rowPtr, <int *>nnzPerRow,
+        <int *>nnzTotal, tol)
```

Comment:
Actually this method has two return values `nnzPerRow` and `nnzTotal`. I think it is not natural to return only one of them.